.circleci/config.yml

```diff
@@ -197,6 +197,8 @@ jobs:
     steps:
       - checkout
       - <<: *restore_dependency_cache_unix
+      # Verifying the release keeps failing due to race condition
+      - run: sleep 10
```

Comment:
There's still a race condition here, it's just a little more likely we won't hit the issue. I've seen npm take minutes before a package is made available.

TLDR; this may not work

Reply:
Lovely... any recommendations on how to actually fix this?

Reply:
I don't have any _good_ recommendations. I think `sleep` is the best bet here. Maybe we just add retries to what you've already got? If the package hasn't been published after 10 tries to verify it, we can just give up and manually verify it.

Reply:
I suppose we could do this:

```sh
VERSION="x.y.z"

function wait_for_publish() {
  npm install "axe-core@$VERSION" < /dev/null
  if [ $? -eq 0 ]; then
    echo "Successfully installed axe"
    return
  else
    sleep 10
    wait_for_publish
  fi
}

wait_for_publish
normal_verify_script # call the existing tool to verify
```